x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/fields/edit_field/edit_field.tsx

```diff
@@ -186,7 +193,28 @@ export const EditField = React.memo(
             </>
           )}
 
-          <EuiFlexGroup justifyContent="flexEnd">
+          <EuiFlexGroup justifyContent="flexEnd" alignItems="center">
+            {isFormModified && (
+              <>
+                <EuiFlexItem>
+                  <EuiText size="s">
+                    <p>
+                      <EuiToolTip
+                        content="Not updating all settings will apply default values to the settings you haven't changed."
```

Comment:
This text needs to be wrapped with `i18n.translate` or `FormattedMessage` so that it can be translated later.